.github/workflows/ci.yml

```diff
@@ -45,15 +45,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '16'
       - name: Install Graphviz
         uses: tlylt/install-graphviz@v1
       - name: Install Java
         uses: actions/setup-java@v3
         with:
           java-version: '11'
           distribution: 'temurin'
-      - run: npm i -g npm@8.3.1
+      - run: npm i -g npm@8.19.3
```

Comment:
Hmmm I thought it might be better to use the same npm version as is currently used for this version of Node.
Should I revert this?

Reply:
sorry, what do you mean by "the same npm version as is currently used for this version of Node"?

Reply:
do you mean you want to keep it at v8, but updating it to the latest build?

Reply:
Like the default version of npm that comes with the [node js](https://nodejs.org/en/download/releases)

Reply:
does that default version of npm come with Node.js 16.19.1 ? If so then there's no need to install and specify npm. The reason why we had that was because the default version of npm for node 14 and below is not v8

Reply:
Ah I see. Thank you I will revert it back

Reply:
Sorry just to confirm, this line can be removed completely because the correct version of npm is already installed right

Reply:
According to the [github actions setup node](https://github.com/actions/setup-node/issues/213#issuecomment-739685886) which is being used here, it will install the corresponding npm version which is 8.19.3 in this case.
I've been using this version of npm and there seems to be no issues so far. 

I will remove it in that case if its ok!